action.yaml

```
@@ -47,6 +47,12 @@ inputs:
     description: Overwrite the function if there is one with the same name.
     default: false
     required: false
+  cpu:
+    description: Set number of CPU cores per function, e.g. 0.8.
```

Haha I suggested something illegal, sorry 🙈 
```suggestion
    description: Set number of CPU cores per function, e.g. 0.5.
```